components/nav/Header.vue

```diff
@@ -262,14 +272,51 @@ export default {
         </button>
 
         <button
-          v-tooltip="t('nav.kubeconfig')"
+          v-tooltip="t('nav.kubeconfigOptions')"
```

Comment:
The dropdown arrow in the header bar is not obvious as to what it does or the fact its a drop down - I'd suggest we remove the drop down and just put two buttons in the header bar - there's plenty of room and they both have icons.